include/zephyr/drivers/led.h

```diff
@@ -330,6 +345,102 @@ static inline int z_impl_led_off(const struct device *dev, uint32_t led)
 	return api->off(dev, led);
 }
 
+/**
+ * @brief Set LED brightness from a led_dt_spec.
+ *
+ * @see led_set_brightness()
+ */
+static inline int led_set_brightness_dt(const struct led_dt_spec *spec,
+					uint8_t value)
```

Comment:
nit: parameters should still be documented

Reply:
cool, added